cli/tests/test_abandon_command.rs

```diff
@@ -50,19 +51,21 @@ fn test_basics() {
     ○ │  [rlv] a
     ├─╯
     ◆  [zzz]
-    "###);
+    [EOF]
+    ");
 
     let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["abandon", "--retain-bookmarks", "d"]);
     insta::assert_snapshot!(stdout, @"");
-    insta::assert_snapshot!(stderr, @r###"
```

Comment:
Is the removal of the `###` in the raw string across many tests expected?

Reply:
Yes, new `insta` version doesn't emit `r###` unless required. It also removes trailing newline if the output doesn't have more than one `\n`.